rust/worker/chroma_config.yaml

```diff
@@ -37,6 +37,7 @@ query_service:
             rate_limiting_policy:
                 count_based_policy:
                     max_concurrent_requests: 15
+                    bandwidth_allocation: [0.9, 0.1]
```

Comment:
it is not immediately clear what priority classes these correspond to
should the config instead be

```
bandwidth_allocations:
	- medium: 0.1
	- high: 0.9
```
?

Reply:
Discussed offline, chose to go with P0, P1, P2... Config here is the same, denoting high to low pri values